visqol-rs/src/variant.rs

```diff
@@ -1,4 +1,6 @@
+use crate::support_vector_regression_model::SupportVectorRegressionModel;
+
 pub enum Variant {
-    Fullband { model_path: String },
+    Fullband { model: SupportVectorRegressionModel },  /// use SupportVectorRegressionModel instead of the file path
```

Comment:
```suggestion
    Fullband { model: SupportVectorRegressionModel }, 
```

This information should go into the commit message, not the source code.